.github/workflows/deploy-dev.yml

```diff
@@ -153,20 +155,22 @@ jobs:
                 fi
             done
           done
-      
-      - name: Deploy to OpenShift Dev
+
+      - name: Set Route Host
         run: |
-          # Create a valid helm release name and route host (lowercase)
           ROUTE_HOST="${{ steps.app_name.outputs.name }}-dev.${{ secrets.APPS_DOMAIN }}"
-          
+          echo "ROUTE_HOST=${ROUTE_HOST}" >> $GITHUB_ENV
```

Comment:
Probeer hier `name=${ROUTE_HOST}` te returnen, het komt beter overeen met de andere code en aan de id van de step is het al duidelijk dat het om een route host gaat